### DISCUSSION TITLE
MySQL schema dump file should be used by default when specific database connection is used

### DISCUSSION THREAD

**Discussion by kwk9892** [laravel/framework#35992] [Ideas]
I used `php artisan schema:dump --prune` to squash migrations

When I try to migrate using `php artisan migrate:fresh` It works where the schema file in `database/schema/` is used

But when I try to migrate using a specific database connection it fails.

Command used: `php artisan migrate:fresh --database=mysql_testing`

I know I can use `--schema-path` to specify the dump file.

In my Testing Modules, I used `use RefreshDatabase` and my database always only have `migrations` table.

Other tables are all dropped.

Is there a way to specify the schema dump file so that it will be used every time a migration runs regardless of the database connection specified.

**Comment by backstageel** [comment]
I was about to ask the same question here
